### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by justinfagnani**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+Occasionally we'll close issues if they appear stale or are too vague - please don't take this personally! Please feel free to re-open issues we've closed if there's something we've missed and they still need to be addressed.
+
+## Pull Requests
+
+Pull requests are greatly appreciated! To ensure a smooth review process, please follow these steps:
+
+ 1. Make sure there's an open issue that the PR addresses. Add "Fixes #(issue number)" to the PR description.
+ 2. Please discuss the general shape of the change ahead of time. This can save much time for reviewers and submitters alike. Many ties there may be ideas on how to handle an issue that are not fully written out, and asking about it will bring out more details.
+ 3. All PRs that change behavior or fix bugs should have new or updated tests.
+ 4. Try to create a set of descriptive commits that each do one focused change. Avoid commits like "oops", and prefer commits like "Added metho foo to Bar".
```
Comment (TimvdLippe) [inline CONTRIBUTING.md]: `Added method foo`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+ 6. If you [allow changes to be committed to your PR branches](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) we can fix some small things in the PR for you, speeding up the review process. This is especially useful if you're new to TypeScript and need help with type annotations.
+
+## Code Style
+
+We follow the [Google JavaScript Style Guide](https://google.github.io/styleguide/jsguide.html), but there are a couple of points worth emphasizing:
+
+ 1. Clear is better than clever. Optimize for simple, readable code first.
+ 2. Prefer longer, more descriptive names, over shorter names. For most variables, minification means we don't pay for extra characters in production.
+ 3. Always err on the side of too many comments. When commenting, "why" is more important than "what".
+ 4. If you're template to add a "what" comment, see if you can't restructure the code and use more descritive names so that the comment is unneccessary.
```
Comment (TimvdLippe) [inline CONTRIBUTING.md]: You are tempting

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+! We welcome and truly appreciate contribution in all forms - issues and pull requests to the [main library](https://github.com/polymer/polymer), issues and pull requests to the [elements the Polymer team maintains](https://github.com/polymerelements), issues and pull requests to one of our many [Polymer-related tools](https://github.com/polymer), and of course we love to hear about any Polymer elements that you build to share with the community!
```
Comment (TimvdLippe) [inline CONTRIBUTING.md]: Should this paragraph be omitted?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
+# Contributing to lit-html
+
+Thank you for your interest in contributing to lit-html!
+
+There are many ways to contribute to lit-html project, and we have many different needs to be addressed. All contributions, from PRs to reports of successful usage, are appreciated and valuable.
+
+## Code of Conduct
+
+We have a (Code of Conduct)[https://github.com/Polymer/polymer/wiki/Code-of-Conduct, please follow it in all interactions with project maintainers and fellow users.
```
Comment (abdonrd) [inline CONTRIBUTING.md]: Close Conduct with the `]`. :)

File: CONTRIBUTING.md (hunk 1 above)
Comment (justinfagnani) [reply CONTRIBUTING.md]: done

File: CONTRIBUTING.md (hunk 2 above)
Comment (justinfagnani) [reply CONTRIBUTING.md]: done

File: CONTRIBUTING.md (hunk 3 above)
Comment (justinfagnani) [reply CONTRIBUTING.md]: done

File: CONTRIBUTING.md (hunk 4 above)
Comment (justinfagnani) [reply CONTRIBUTING.md]: done

**Pull request merged**